### DISCUSSION TITLE
Page flipping (time lag) after activating the auto straighten feature.

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#8984] [Q&A]
So, I don't think it's a bug that's why I'm asking here.
After activating the auto straighten feature flipping pages is a bit (slow), it takes several seconds (up to 8?) to flip the page, and it happens it both flipping forward and backward.
By the way I'm using an android tablet (Samsung galaxy Tab T-295) and Samsung galaxy A02, and the pdfs I'm having this with is a scanned book so it's actually images converted to PDF and some pages are really wrapped.

Is that normal?

**Comment by Frenzie** [comment]
> Is that normal?

For some combination of a really, really heavy file and/or a slow device, sure.

**Reply by Zeyadas** [reply]
Thank you.
